examples/3_Liu_Copper_OrgSyn/example_liu.ipynb

```diff
@@ -67,7 +70,9 @@
     "    !git clone https://github.com/Open-Reaction-Database/ord-schema.git\n",
     "    %cd ord-schema\n",
     "    !python setup.py install\n",
-    "    sys.path.append('/usr/local/lib/python3.7/site-packages/')"
+    "    sys.path.append('/usr/local/lib/python3.7/site-packages/')\n",
```

Comment:
Probably better as:

```shell
%cd ord-schema
!pip install -r requirements.txt
!python setup.py install
```

I'm very hesitant to hardcode the PATH here; environment variables should be sufficient.